<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>btree_add</class>
 <widget class="QDialog" name="btree_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>138</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="editName"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editSurname"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editPatrnymic"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editIdentNumb"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editRegistrDate"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="Name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Surname">
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Patronymic">
        <property name="text">
         <string>Patronymic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="IdentNumb">
        <property name="text">
         <string>IdentNumb</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RegistrDate">
        <property name="text">
         <string>RegistrDate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
